tests/pfc/conftest.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from tests.conftest import generate_port_lists, generate_priority_lists
+
+@pytest.fixture(autouse=True, scope="module")
+def rand_one_oper_up_intf(request):
+    """
+    Fixture that randomly selects one oper up interface
+
+    Args:
+        request (object): pytest request object
+
+    Yields:
+        interface (str): string containing 'hostname|selected intf'
+
+    """
+    oper_up_intfs = generate_port_lists(request, "oper_up_ports")
+    if oper_up_intfs:
+        yield random.sample(oper_up_intfs, 1)[0]
+    else:
+        yield 'unknown|unknown'
+
+@pytest.fixture(autouse=True, scope="module")
+def rand_lossless_prio(request):
```

Comment:
I have defined rand_one_dut_portname_oper_up and rand_one_dut_lossless_prio in conftest.py in another PR (https://github.com/Azure/sonic-mgmt/pull/2719/files). Can you reuse those fixtures?

Reply:
Since that PR isn't merged, I am planning to  keep this as is. Will come back with another PR to reuse the fixtures once that is merged